apps/docs/src/app/core/component-docs/dialog/dialog-docs.component.html

```diff
@@ -0,0 +1,194 @@
+        <li><code>DialogRef.dismiss()</code> - dismissing the Dialog window</li>
+        <li><code>DialogRef.loading()</code> - displaying the dialog in loading state</li>
+    </ul>
+</description>
+<component-example>
+    <fd-dialog-state-example></fd-dialog-state-example>
+</component-example>
+<code-example [exampleFiles]="stateDialog"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogConfiguration'" [componentName]="'dialog'">
+    Dialog configuration
+</fd-docs-section-title>
+<description>
+    Dialog behavior can be configured using <code>DialogConfig</code> object passed as an argument to <code>DialogService.open()</code>
+    method. Passing <code>DialogConfig</code> overrides specified behaviours of the Dialog component.
+</description>
+<component-example>
+    <fd-dialog-configuration-example></fd-dialog-configuration-example>
+</component-example>
+<code-example [exampleFiles]="customConfiguration"></code-example>
+
+<description>
+    Default values used by the Dialog can be specified by providing <code>DIALOG_DEFAULT_CONFIG</code> token value.<br/><br/>
+    <code>
+        @NgModule({{'{'}}
+            provide: [
+                {{'{'}}provide: DIALOG_DEFAULT_CONFIG, useValue: customConfig{{'}'}}
+            ]
+        {{'}'}}
+    </code>
+</description>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogPosition'" [componentName]="'dialog'">
+    Dialog positioning
+</fd-docs-section-title>
+<description>
+    The placement of the Dialog on the screen can be customized through the <code>DialogConfig.position</code> property.
+</description>
+<component-example>
+    <fd-dialog-position-example></fd-dialog-position-example>
+</component-example>
+<code-example [exampleFiles]="dialogPosition"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'dialogMobile'" [componentName]="'dialog'">
+    Dialog on mobile devices
+</fd-docs-section-title>
+<description>
+    To display Dialog component in mobile mode set use <code>DialogConfig.mobile</code> property.
+    Dialog on mobile devices takes full viewport and adjusts its header and footer. To create additional spacing around
+    Dialog use <code>DialogConfig.mobileOuterSpacing</code> property.
+    <br/><br/>
+    To achieve full-screen dialog without mobile mode features use <code>DialogConfig.fullScreen</code> property.
+</description>
+<component-example>
+    <fd-dialog-mobile-example></fd-dialog-mobile-example>
+</component-example>
+<code-example [exampleFiles]="dialogMobile"></code-example>
+
+
+<separator></separator>
+
+<fd-docs-section-title [id]="'complexDialog'" [componentName]="'dialog'">
+    Complex Dialog
+</fd-docs-section-title>
+<description>
+    To create more complex Dialogs it is possible to provide custom header, subheader and footer templates.<br/>
+    To provide custom template create element with on of the following directives:
```

Comment:
To provide these custom templates, create an element with one of the following directives.